source/nodejs/adaptivecards-templating/src/template-engine.ts

```diff
@@ -3,17 +3,19 @@
 import * as AEL from "adaptive-expressions";
 
 class EvaluationContext {
```

Comment:
Please update the doc too. You can create a bug and separate PR for the document